tests/integration/test_kinesis.py

```diff
@@ -118,6 +114,7 @@ def test_subscribe_to_shard_with_sequence_number_as_iterator(self):
         result = client.create_stream(StreamName=stream_name, ShardCount=1)
         sleep(1)
         result = client.register_stream_consumer(StreamARN=stream_arn, ConsumerName='c1')['Consumer']
+        sleep(1)
```

Comment:
kinesis-mock applies latency to RegisterStreamConsumer so a sleep is needed here.